### ISSUE TITLE
Komga not starting with error

### ISSUE THREAD

**Issue by Salvora** [gotson/komga#1048]
### Steps to reproduce

1. Restart the container.
2. Try to access the webui
3. Check the logs

### Expected behavior

Should start

### Actual behavior

Errors

### Logs

<details>
  <summary>Show logs</summary>

```
2023-01-16 19:53:16.356  INFO 1 --- [main] org.gotson.komga.ApplicationKt           : Starting ApplicationKt using Java 11.0.17 on ed9b0459dbe1 with PID 1 (/app/BOOT-INF/classes started by ? in /app)
2023-01-16 19:53:16.437  INFO 1 --- [main] org.gotson.komga.ApplicationKt           : No active profile set, falling back to 1 default profile: "default"
2023-01-16 19:53:36.587  INFO 1 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8080 (https)
2023-01-16 19:53:36.634  INFO 1 --- [main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2023-01-16 19:53:36.635  INFO 1 --- [main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.68]
2023-01-16 19:53:37.180  INFO 1 --- [main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2023-01-16 19:53:37.181  INFO 1 --- [main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 20262 ms
2023-01-16 19:53:39.275  INFO 1 --- [main] com.zaxxer.hikari.HikariDataSource       : SqliteUdfPool - Starting...
2023-01-16 19:53:40.651  INFO 1 --- [main] com.zaxxer.hikari.HikariDataSource       : SqliteUdfPool - Start completed.
2023-01-16 19:53:42.330  INFO 1 --- [main] o.s.b.a.e.web.ServletEndpointRegistrar   : Registered '/actuator/jolokia' to jolokia-actuator-endpoint
2023-01-16 19:53:42.510  INFO 1 --- [main] io.hawt.HawtioContextListener            : Initialising hawtio services
2023-01-16 19:53:43.091  INFO 1 --- [main] io.hawt.system.ConfigManager             : Configuration will be discovered via system properties
2023-01-16 19:53:43.098  INFO 1 --- [main] io.hawt.jmx.JmxTreeWatcher               : Welcome to Hawtio 2.16.1
2023-01-16 19:53:43.161  INFO 1 --- [main] i.h.w.auth.AuthenticationConfiguration   : Starting hawtio authentication filter, JAAS realm: "karaf" authorized role(s): "admin,manager,viewer" role principal classes: ""
2023-01-16 19:53:43.172  INFO 1 --- [main] io.hawt.web.auth.LoginRedirectFilter     : Hawtio loginRedirectFilter is using 1800 sec. HttpSession timeout
2023-01-16 19:53:44.310  INFO 1 --- [main] org.apache.activemq.artemis.core.server  : AMQ221000: live Message Broker is starting with configuration Broker Configuration (clustered=false,journalDirectory=/config/artemis/journal,bindingsDirectory=/config/artemis/bindings,largeMessagesDirectory=/config/artemis/largemessages,pagingDirectory=/config/artemis/paging)
2023-01-16 19:53:45.355  INFO 1 --- [main] org.apache.activemq.artemis.core.server  : AMQ221013: Using NIO Journal
2023-01-16 19:53:46.347  INFO 1 --- [main] org.apache.activemq.artemis.core.server  : AMQ221057: Global Max Size is being adjusted to 1/2 of the JVM max size (-Xmx). being defined as 2,147,483,648
2023-01-16 19:53:46.708  INFO 1 --- [main] org.apache.activemq.artemis.core.server  : AMQ221043: Protocol module found: [artemis-server]. Adding protocol support for: CORE
2023-01-16 19:53:47.039  INFO 1 --- [main] org.apache.activemq.artemis.core.server  : AMQ221034: Waiting indefinitely to obtain live lock
2023-01-16 19:53:47.040  INFO 1 --- [main] org.apache.activemq.artemis.core.server  : AMQ221035: Live Server Obtained live lock
2023-01-16 19:56:44.017  INFO 1 --- [main] org.gotson.komga.ApplicationKt           : Starting ApplicationKt using Java 11.0.17 on 4b28d3fd3cf8 with PID 1 (/app/BOOT-INF/classes started by ? in /app)
2023-01-16 19:56:44.038  INFO 1 --- [main] org.gotson.komga.ApplicationKt           : No active profile set, falling back to 1 default profile: "default"
2023-01-16 19:57:10.811  INFO 1 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8080 (https)
2023-01-16 19:57:10.882  INFO 1 --- [main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2023-01-16 19:57:10.883  INFO 1 --- [main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.68]
2023-01-16 19:57:11.683  INFO 1 --- [main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2023-01-16 19:57:11.684  INFO 1 --- [main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 27030 ms
2023-01-16 19:57:13.955  INFO 1 --- [main] com.zaxxer.hikari.HikariDataSource       : SqliteUdfPool - Starting...
2023-01-16 19:57:15.727  INFO 1 --- [main] com.zaxxer.hikari.HikariDataSource       : SqliteUdfPool - Start completed.
2023-01-16 19:57:18.127  INFO 1 --- [main] o.s.b.a.e.web.ServletEndpointRegistrar   : Registered '/actuator/jolokia' to jolokia-actuator-endpoint
2023-01-16 19:57:18.181  INFO 1 --- [main] io.hawt.HawtioContextListener            : Initialising hawtio services
2023-01-16 19:57:18.903  INFO 1 --- [main] io.hawt.system.ConfigManager             : Configuration will be discovered via system properties
2023-01-16 19:57:18.908  INFO 1 --- [main] io.hawt.jmx.JmxTreeWatcher               : Welcome to Hawtio 2.16.1
2023-01-16 19:57:18.972  INFO 1 --- [main] i.h.w.auth.AuthenticationConfiguration   : Starting hawtio authentication filter, JAAS realm: "karaf" authorized role(s): "admin,manager,viewer" role principal classes: ""
2023-01-16 19:57:18.981  INFO 1 --- [main] io.hawt.web.auth.LoginRedirectFilter     : Hawtio loginRedirectFilter is using 1800 sec. HttpSession timeout
2023-01-16 19:57:20.226  INFO 1 --- [main] org.apache.activemq.artemis.core.server  : AMQ221000: live Message Broker is starting with configuration Broker Configuration (clustered=false,journalDirectory=/config/artemis/journal,bindingsDirectory=/config/artemis/bindings,largeMessagesDirectory=/config/artemis/largemessages,pagingDirectory=/config/artemis/paging)
2023-01-16 19:57:20.436  INFO 1 --- [main] org.apache.activemq.artemis.core.server  : AMQ221013: Using NIO Journal
2023-01-16 19:57:22.065  INFO 1 --- [main] org.apache.activemq.artemis.core.server  : AMQ221057: Global Max Size is being adjusted to 1/2 of the JVM max size (-Xmx). being defined as 4,294,967,296
2023-01-16 19:57:22.153  INFO 1 --- [main] org.apache.activemq.artemis.core.server  : AMQ221043: Protocol module found: [artemis-server]. Adding protocol support for: CORE
2023-01-16 19:57:22.690  INFO 1 --- [main] org.apache.activemq.artemis.core.server  : AMQ221034: Waiting indefinitely to obtain live lock
2023-01-16 19:57:22.691  INFO 1 --- [main] org.apache.activemq.artemis.core.server  : AMQ221035: Live Server Obtained live lock
2023-01-16 20:00:20.167  WARN 1 --- [Thread-0 (-scheduled-threads)] o.a.a.a.utils.critical.CriticalMeasure   : Component org.apache.activemq.artemis.core.persistence.impl.journal.JournalStorageManager is expired on path 0
2023-01-16 20:00:20.170  INFO 1 --- [Thread-0 (-scheduled-threads)] org.apache.activemq.artemis.core.server  : AMQ224107: The Critical Analyzer detected slow paths on the broker.  It is recommended that you enable trace logs on org.apache.activemq.artemis.utils.critical while you troubleshoot this issue. You should disable the trace logs when you have finished troubleshooting.
2023-01-16 20:00:20.171  WARN 1 --- [Thread-0 (-scheduled-threads)] org.apache.activemq.artemis.core.server  : AMQ224081: The component org.apache.activemq.artemis.core.persistence.impl.journal.JournalStorageManager@6c49ea0c is not responsive
2023-01-16 20:00:20.617  WARN 1 --- [Thread-0 (-scheduled-threads)] org.apache.activemq.artemis.core.server  : AMQ222199: Thread dump: *******************************************************************************
Complete Thread dump 
"main" Id=1 RUNNABLE (in native)
	at java.base@11.0.17/sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at java.base@11.0.17/sun.nio.ch.FileDispatcherImpl.read(Unknown Source)
	at java.base@11.0.17/sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at java.base@11.0.17/sun.nio.ch.IOUtil.read(Unknown Source)
	at java.base@11.0.17/sun.nio.ch.FileChannelImpl.read(Unknown Source)
	-  locked java.lang.Object@19eee953
	at org.apache.activemq.artemis.core.io.nio.NIOSequentialFile.read(NIOSequentialFile.java:291)
	-  locked org.apache.activemq.artemis.core.io.nio.NIOSequentialFile@29998c7
	at org.apache.activemq.artemis.core.io.nio.NIOSequentialFile.read(NIOSequentialFile.java:239)
	at org.apache.activemq.artemis.core.journal.impl.JournalImpl.readJournalFile(JournalImpl.java:601)
	at org.apache.activemq.artemis.core.journal.impl.JournalImpl.load(JournalImpl.java:2140)
	-  locked org.apache.activemq.artemis.core.journal.impl.JournalImpl@617d032c
	at org.apache.activemq.artemis.core.journal.impl.JournalImpl.load(JournalImpl.java:2433)
	-  locked org.apache.activemq.artemis.core.journal.impl.JournalImpl@617d032c
	at org.apache.activemq.artemis.core.journal.impl.JournalImpl.load(JournalImpl.java:1739)
	-  locked org.apache.activemq.artemis.core.journal.impl.JournalImpl@617d032c
	at org.apache.activemq.artemis.core.journal.Journal.load(Journal.java:278)
	at org.apache.activemq.artemis.core.persistence.impl.journal.AbstractJournalStorageManager.loadMessageJournal(AbstractJournalStorageManager.java:873)
	at org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl.loadJournals(ActiveMQServerImpl.java:3570)
	at org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl.initialisePart2(ActiveMQServerImpl.java:3233)
	-  locked org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl@53cfd343
	at org.apache.activemq.artemis.core.server.impl.LiveOnlyActivation.run(LiveOnlyActivation.java:76)
	at org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl.internalStart(ActiveMQServerImpl.java:655)
	at org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl.start(ActiveMQServerImpl.java:568)
	-  locked org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl@53cfd343
	at org.apache.activemq.artemis.core.server.embedded.EmbeddedActiveMQ.start(EmbeddedActiveMQ.java:116)
	at java.base@11.0.17/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base@11.0.17/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base@11.0.17/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base@11.0.17/java.lang.reflect.Method.invoke(Unknown Source)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeCustomInitMethod(AbstractAutowireCapableBeanFactory.java:1930)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1872)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1800)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:620)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335)
	at org.springframework.beans.factory.support.AbstractBeanFactory$$Lambda$255/0x00000008401d6440.getObject(Unknown Source)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234)
	-  locked java.util.concurrent.ConcurrentHashMap@3d051c18
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.getBeansOfType(DefaultListableBeanFactory.java:673)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.getBeansOfType(DefaultListableBeanFactory.java:661)
	at org.springframework.boot.autoconfigure.jms.artemis.ArtemisConnectionFactoryFactory.startEmbeddedJms(ArtemisConnectionFactoryFactory.java:73)
	at org.springframework.boot.autoconfigure.jms.artemis.ArtemisConnectionFactoryFactory.createConnectionFactory(ArtemisConnectionFactoryFactory.java:61)
	at org.springframework.boot.autoconfigure.jms.artemis.ArtemisConnectionFactoryConfiguration$SimpleConnectionFactoryConfiguration.createConnectionFactory(ArtemisConnectionFactoryConfiguration.java:81)
	at org.springframework.boot.autoconfigure.jms.artemis.ArtemisConnectionFactoryConfiguration$SimpleConnectionFactoryConfiguration.cachingJmsConnectionFactory(ArtemisConnectionFactoryConfiguration.java:66)
	at java.base@11.0.17/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base@11.0.17/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base@11.0.17/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base@11.0.17/java.lang.reflect.Method.invoke(Unknown Source)
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:154)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:653)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:638)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateUsingFactoryMethod(AbstractAutowireCapableBeanFactory.java:1352)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1195)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:582)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335)
	at org.springframework.beans.factory.support.AbstractBeanFactory$$Lambda$255/0x00000008401d6440.getObject(Unknown Source)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234)
	-  locked java.util.concurrent.ConcurrentHashMap@3d051c18
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208)
	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:276)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1391)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1311)
	at org.springframework.beans.factory.support.ConstructorResolver.resolveAutowiredArgument(ConstructorResolver.java:887)
	at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:791)
	at org.springframework.beans.factory.support.ConstructorResolver.autowireConstructor(ConstructorResolver.java:229)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireConstructor(AbstractAutowireCapableBeanFactory.java:1372)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1222)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:582)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335)
	at org.springframework.beans.factory.support.AbstractBeanFactory$$Lambda$255/0x00000008401d6440.getObject(Unknown Source)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234)
	-  locked java.util.concurrent.ConcurrentHashMap@3d051c18
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:955)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:918)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:583)
	-  locked java.lang.Object@7314f4df
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:147)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:734)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:408)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:308)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1306)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1295)
	at org.gotson.komga.ApplicationKt.main(Application.kt:16)
	at java.base@11.0.17/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base@11.0.17/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base@11.0.17/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base@11.0.17/java.lang.reflect.Method.invoke(Unknown Source)
	at app//org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49)
	at app//org.springframework.boot.loader.Launcher.launch(Launcher.java:108)
	at app//org.springframework.boot.loader.Launcher.launch(Launcher.java:58)
	at app//org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:65)


"Reference Handler" Id=2 RUNNABLE
	at java.base@11.0.17/java.lang.ref.Reference.waitForReferencePendingList(Native Method)
	at java.base@11.0.17/java.lang.ref.Reference.processPendingReferences(Unknown Source)
	at java.base@11.0.17/java.lang.ref.Reference$ReferenceHandler.run(Unknown Source)


"Finalizer" Id=3 WAITING on java.lang.ref.ReferenceQueue$Lock@7a79edf2
	at java.base@11.0.17/java.lang.Object.wait(Native Method)
	-  waiting on java.lang.ref.ReferenceQueue$Lock@7a79edf2
	at java.base@11.0.17/java.lang.ref.ReferenceQueue.remove(Unknown Source)
	at java.base@11.0.17/java.lang.ref.ReferenceQueue.remove(Unknown Source)
	at java.base@11.0.17/java.lang.ref.Finalizer$FinalizerThread.run(Unknown Source)


"Signal Dispatcher" Id=4 RUNNABLE


"Common-Cleaner" Id=10 TIMED_WAITING on java.lang.ref.ReferenceQueue$Lock@57816c35
	at java.base@11.0.17/java.lang.Object.wait(Native Method)
	-  waiting on java.lang.ref.ReferenceQueue$Lock@57816c35
	at java.base@11.0.17/java.lang.ref.ReferenceQueue.remove(Unknown Source)
	at java.base@11.0.17/jdk.internal.ref.CleanerImpl.run(Unknown Source)
	at java.base@11.0.17/java.lang.Thread.run(Unknown Source)
	at java.base@11.0.17/jdk.internal.misc.InnocuousThread.run(Unknown Source)


"logback-1" Id=12 WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@42f15f01
	at java.base@11.0.17/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@42f15f01
	at java.base@11.0.17/java.util.concurrent.locks.LockSupport.park(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor.getTask(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base@11.0.17/java.lang.Thread.run(Unknown Source)


"SqliteUdfPool housekeeper" Id=16 TIMED_WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@1617f98c
	at java.base@11.0.17/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@1617f98c
	at java.base@11.0.17/java.util.concurrent.locks.LockSupport.parkNanos(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor.getTask(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base@11.0.17/java.lang.Thread.run(Unknown Source)


"Catalina-utility-1" Id=17 WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@233480c1
	at java.base@11.0.17/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@233480c1
	at java.base@11.0.17/java.util.concurrent.locks.LockSupport.park(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor.getTask(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base@11.0.17/java.lang.Thread.run(Unknown Source)


"Catalina-utility-2" Id=18 TIMED_WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@233480c1
	at java.base@11.0.17/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@233480c1
	at java.base@11.0.17/java.util.concurrent.locks.LockSupport.parkNanos(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor.getTask(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base@11.0.17/java.lang.Thread.run(Unknown Source)


"container-0" Id=19 TIMED_WAITING
	at java.base@11.0.17/java.lang.Thread.sleep(Native Method)
	at org.apache.catalina.core.StandardServer.await(StandardServer.java:566)
	at org.springframework.boot.web.embedded.tomcat.TomcatWebServer$1.run(TomcatWebServer.java:197)


"Thread-0 (-scheduled-threads)" Id=22 RUNNABLE
	at java.management@11.0.17/sun.management.ThreadImpl.dumpThreads0(Native Method)
	at java.management@11.0.17/sun.management.ThreadImpl.dumpAllThreads(Unknown Source)
	at java.management@11.0.17/sun.management.ThreadImpl.dumpAllThreads(Unknown Source)
	at org.apache.activemq.artemis.utils.ThreadDumpUtil.threadDump(ThreadDumpUtil.java:47)
	at org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl.threadDump(ActiveMQServerImpl.java:1140)
	at org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl.lambda$initializeCriticalAnalyzer$2(ActiveMQServerImpl.java:772)
	at org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl$$Lambda$893/0x0000000840736440.run(Unknown Source)
	at org.apache.activemq.artemis.utils.critical.CriticalAnalyzerImpl.fireActions(CriticalAnalyzerImpl.java:155)
	at org.apache.activemq.artemis.utils.critical.CriticalAnalyzerImpl.check(CriticalAnalyzerImpl.java:140)
	at org.apache.activemq.artemis.utils.critical.CriticalAnalyzerImpl$1.run(CriticalAnalyzerImpl.java:53)
	at org.apache.activemq.artemis.core.server.ActiveMQScheduledComponent.runForExecutor(ActiveMQScheduledComponent.java:313)
	at org.apache.activemq.artemis.core.server.ActiveMQScheduledComponent.bookedRunForScheduler(ActiveMQScheduledComponent.java:328)
	at org.apache.activemq.artemis.core.server.ActiveMQScheduledComponent.runForScheduler(ActiveMQScheduledComponent.java:339)
	at org.apache.activemq.artemis.core.server.ActiveMQScheduledComponent.lambda$start$0(ActiveMQScheduledComponent.java:166)
	at org.apache.activemq.artemis.core.server.ActiveMQScheduledComponent$$Lambda$892/0x0000000840736040.run(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at org.apache.activemq.artemis.utils.ActiveMQThreadFactory$1.run(ActiveMQThreadFactory.java:118)

	Number of locked synchronizers = 1
	- java.util.concurrent.ThreadPoolExecutor$Worker@4552f905


"activemq-buffer-timeout" Id=23 WAITING on java.util.concurrent.Semaphore$NonfairSync@7f88b056
	at java.base@11.0.17/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.Semaphore$NonfairSync@7f88b056
	at java.base@11.0.17/java.util.concurrent.locks.LockSupport.park(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.locks.AbstractQueuedSynchronizer.parkAndCheckInterrupt(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.locks.AbstractQueuedSynchronizer.doAcquireSharedInterruptibly(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.locks.AbstractQueuedSynchronizer.acquireSharedInterruptibly(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.Semaphore.acquire(Unknown Source)
	at org.apache.activemq.artemis.core.io.buffer.TimedBuffer$CheckTimer.run(TimedBuffer.java:467)
	at java.base@11.0.17/java.lang.Thread.run(Unknown Source)


"Thread-0 (ActiveMQ-scheduled-threads)" Id=24 TIMED_WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@37c9f1f1
	at java.base@11.0.17/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@37c9f1f1
	at java.base@11.0.17/java.util.concurrent.locks.LockSupport.parkNanos(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor.getTask(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at org.apache.activemq.artemis.utils.ActiveMQThreadFactory$1.run(ActiveMQThreadFactory.java:118)


"Thread-1 (ActiveMQ-scheduled-threads)" Id=25 WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@37c9f1f1
	at java.base@11.0.17/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@37c9f1f1
	at java.base@11.0.17/java.util.concurrent.locks.LockSupport.park(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor.getTask(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at org.apache.activemq.artemis.utils.ActiveMQThreadFactory$1.run(ActiveMQThreadFactory.java:118)


"Thread-2 (ActiveMQ-scheduled-threads)" Id=26 WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@37c9f1f1
	at java.base@11.0.17/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@37c9f1f1
	at java.base@11.0.17/java.util.concurrent.locks.LockSupport.park(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor.getTask(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at org.apache.activemq.artemis.utils.ActiveMQThreadFactory$1.run(ActiveMQThreadFactory.java:118)


"Thread-3 (ActiveMQ-scheduled-threads)" Id=27 WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@37c9f1f1
	at java.base@11.0.17/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@37c9f1f1
	at java.base@11.0.17/java.util.concurrent.locks.LockSupport.park(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor.getTask(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at org.apache.activemq.artemis.utils.ActiveMQThreadFactory$1.run(ActiveMQThreadFactory.java:118)


"Thread-4 (ActiveMQ-scheduled-threads)" Id=28 WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@37c9f1f1
	at java.base@11.0.17/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@37c9f1f1
	at java.base@11.0.17/java.util.concurrent.locks.LockSupport.park(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor.getTask(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base@11.0.17/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at org.apache.activemq.artemis.utils.ActiveMQThreadFactory$1.run(ActiveMQThreadFactory.java:118)


===============================================================================
End Thread dump 
*******************************************************************************
```
</details>

[komga.log](https://github.com/gotson/komga/files/10427888/komga.log)

### Komga version

0.158.0

### Operating system

Synology NAS Docker

### Other details

My database sqlite file is 2gb.
I tried allocating more memory (from 4gb to 8gb)
The comic files are on the same NAS.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have checked the [FAQ](https://komga.org/faq/).
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
The database is not available: `Caused by: java.sql.SQLTransientConnectionException: SqliteUdfPool - Connection is not available, request timed out after 30000ms.`

1. Did it ever work before?
2. Can you provide the Docker configuration, especially volumes?

**Comment by Salvora** [comment]
yes, it used to work before. It has been a while since I used it. I noticed this after updating the container.

```yaml
version: '3.9'
services:
  komga:
    image: gotson/komga
    container_name: Komga
    labels:
      - "com.centurylinklabs.watchtower.enable"
    volumes:
      - /volume1/docker/Komga/config:/config
      - /volume1/Gulibu/Gulibu/Comics:/data/Comics
      - /volume1/docker/Komga/timezone:/etc/timezone:ro
      - /volume1/docker/SSL:/mnt/SSL:ro
    ports:
      - 11200:8080
    user: "1034:100"
    environment:
      - JAVA_TOOL_OPTIONS="-Xmx8g"
      - TZ=Europe/Istanbul

    restart: unless-stopped
```

**application.yml:**

```yaml
# Only keep lines that are relevant to you!
# Lines starting with # are comments
# Make sure indentation is correct (2 spaces at every indentation level), yaml is very sensitive!
komga:
  libraries-scan-cron: "-" #"0 */15 * * * ?" #periodic scan every 15 minutes use "-" for disabling it.
  libraries-scan-startup: true #scan libraries at startup
  libraries-scan-directory-exclusions: #patterns to exclude directories from the scan
    - "#recycle" #synology NAS recycle bin
    - "@eaDir"   #synology NAS index/metadata folders
  remember-me:
    key: "vLhorr245X9R1ILLXTTt43622455vcfdWW22dsfg5SDGgdsg3412dfghGGDSAS" #required to activate the remember-me auto-login via cookies
    validity: 365d #validity of the cookie in seconds, here 1 month
  session-timeout: 30d
  cors:
    allowed-origins:
      - https://192.168.1.45
      - https://192.168.50.45
  delete-empty-collections: true #A boolean indicating if Komga should delete empty collections after a scan.
  delete-empty-read-lists: true

server:
  ssl:
    key-store: /mnt/SSL/komga.p12
    key-store-password: root
    key-store-type: pkcs12
    key-alias: komga
```

**Comment by gotson** [comment]
The logs you provided are not from the start of the container. Can you provide some logs from startup until the symptom?

**Comment by Salvora** [comment]
New logs are attached with the start sequence.
[komga.log](https://github.com/gotson/komga/files/10438095/komga.log)

**Comment by gotson** [comment]
Could you set the following environment variable and get some logs from startup again?

`LOGGING_LEVEL_ORG_APACHE_ACTIVEMQ_ARTEMIS_UTILS_CRITICAL=TRACE`

**Comment by Salvora** [comment]
[komga.log](https://github.com/gotson/komga/files/10441971/komga.log)
Log are attached with env.

**Comment by gotson** [comment]
I suspect you have some issues with the mounted `/config` volume.

I remember similar issues on Synology where indexing was done on the folder, which caused similar problems. Can you fiddle in the Synology UI/config and see if you can remove any indexing job on the `/volume1/docker/Komga/config` folder?

**Comment by Salvora** [comment]
I have indexing totally disabled on my synology at all times on all folders. Is it possible for this issue to be related with the low available free space on the disk? I have about 100gb free usable space left on the NAS. It should be more than enough though.

**Comment by gotson** [comment]
> I have indexing totally disabled on my synology at all times on all folders. Is it possible for this issue to be related with the low available free space on the disk? I have about 100gb free usable space left on the NAS. It should be more than enough though.

you would most likely get different exceptions in case of low disk space
